WorldMoodTracker/js/utils/index.js

```diff
@@ -8,3 +10,19 @@ export function containsObject(obj, list)
     }
     return false;
 }
+
+let countryReverseIndex = {};
+
+for (let code in iso3166.codes) {
+    let codeAlpha3 = iso3166.codes[code];
+    countryReverseIndex[codeAlpha3] = code;
+}
+
+export function countryCodeConverter(code)
+{
```

Comment:
The numbers and colours would change as they are set as the `state` for the `WorldMap` component. Since most of the Tweets collected are from recent times, the numbers will not vary much and changes in colour would be hard to notice.